web3/eth/eth.py

```diff
@@ -224,7 +224,7 @@ def fee_history(
         self,
         block_count: int,
         newest_block: Union[BlockParams, BlockNumber],
-        reward_percentiles: Optional[List[float]] = None,
+        reward_percentiles: Optional[List[float]] = [],
```

Comment:
instead, you can do `reward_percentiles = reward_percentiles or []` as the first line in the function's implementation

Reply:
Thanks for the heads up! Updated.